src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -132,6 +174,19 @@
   }
 }
 
+.pf-c-context-selector__toggle-button-icon {
```

Comment:
Is this element used anywhere?

Reply:
If this isn't being used, can you remove this block?

Reply:
Looks like this block is still here - I think it's safe to remove since I don't see an element with this class used anywhere in the component?